components/FormikFields/PaperInputPicker/index.js

```diff
@@ -56,6 +55,7 @@ const PaperInputPicker = ({
       )}
       {fieldType === 'select' && (
         <View>
+          <Title>{label}</Title>
```

Comment:
This was a hotfix to add labels for select forms (really should be in a different PR)